pkgs/applications/misc/notesnook/default.nix

```diff
@@ -1,24 +1,27 @@
 { lib, stdenv, appimageTools, fetchurl, _7zz }:
 
 let
-  pname = "notesnook";
-  version = "3.0.8";
-
   inherit (stdenv.hostPlatform) system;
   throwSystem = throw "Unsupported system: ${system}";
 
+  pname = "notesnook";
+  version = "3.0.13";
+
+
```

Comment:
Could you please put these lines back to the top?

And please, squash your commits into a single one named `notesnook: 3.0.8 -> 3.0.13`, to follow the [commit conventions](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#commit-conventions) for packages.

Reply:
Yeah done

Reply:
That's better but you also need to remove descriptions of squashed commits.

Reply:
Check now